lib/autokey/macro.py

```diff
@@ -95,23 +111,23 @@ def _can_process(self, token):
             return False
 
     def _get_args(self, token):
-        print(token)
-        l = token[:-1].split(' ')
-        ret = {}
-
-        if len(l) > 1:
-            for arg in l[1:]:
-                key, val = arg.split('=', 1)
-                ret[key] = val
-
-        for k, v in self.ARGS:
-            if k not in ret:
-                raise Exception("Missing mandatory argument '{}' for macro '{}'".format(k, self.ID))
```

Comment:
Since this PR is against `develop`, changing the Exception type to ValueError is ok. This is something I wanted to do anyways, so it’s nice to see it done.